dirsrvtests/tests/suites/webui/backup/backup_test.py

```diff
@@ -0,0 +1,53 @@
+# --- BEGIN COPYRIGHT BLOCK ---
+# Copyright (C) 2023 Red Hat, Inc.
+# All rights reserved.
+#
+# License: GPL (version 3 or any later version).
+# See LICENSE for details.
+# --- END COPYRIGHT BLOCK ---
+#
+import time
+import subprocess
+import pytest
+
+from lib389.cli_idm.account import *
+from lib389.tasks import *
+from lib389.utils import *
+from lib389.topologies import topology_st
+from .. import setup_page, check_frame_assignment, setup_login
+
+pytestmark = pytest.mark.skipif(os.getenv('WEBUI') is None, reason="These tests are only for WebUI environment")
+pytest.importorskip('playwright')
+
+SERVER_ID = 'standalone1'
+
+
```

Comment:
Let's add this line to mark it as 'expected to fail' until the bz is fixed:

@pytest.mark.xfail(reason="Will fail because of bz2189181")

Otherwise LGTM.